### ISSUE TITLE
Ensure aria-selected generated as output of grid_for

### ISSUE THREAD

**Issue by sidnc86** [techvision/waiable#5]
Enable aria-selected property on the grid cell so that it announces its state to screen reader user. For this ensure that markup is generated as follows
```html
<div role=”grid” tabindex=”-1”>
  <div role=”row” tabindex=”-1”>
    <div role=”gridcell” tabindex=”0” aria-selected=”true”>
      ...
    </div>
    <div role=”gridcell” tabindex=”-1”>
      ...
    </div>
  </div>
  ....
</div>
```
Note that care is taken to assign aria-selected=”true” only to one gridcell that will be focused when user navigates the grid by pressing tab keys.
This should be generated as part of grid_for method made available from the gem.

**Comment by sidnc86** [comment]
Closing as focus is now switched towards basic content accessibility instead of building rich widgets.
